### ISSUE TITLE
Description mysql failed to restart unexpectedly

### ISSUE THREAD

**Issue by lizhongxuan** [bitpoke/mysql-operator#894]
When the mysql disk is full, the pod will restart. If the mysql redo log is too large, it will take a long time to redo. The heartbeat detection pod will cause the entire image to restart. Keep repeating

**Comment by drivebyer** [comment]
Maybe should increase the pvc size?

**Comment by lizhongxuan** [comment]
> 

No, the disk is fine. mysql log redo conflicts with the heartbeat mechanism. I did the following actions to restore mysql: 1. Close the operator; 2 Change the detection times to 9999; 3. After mysql recovers, restore operator.

**Comment by lizhongxuan** [comment]
When mysql reworks the log for too long,mysql does not recover, triggering a pod restart.

**Comment by drivebyer** [comment]
I understand you. BTW, how do you config the detection time, where is the configuration?
